### PR TITLE
apps: add letmecreateiot submodule

### PR DESCRIPTION
**Pull request by mtusnio**
This connects to #47 

Adds LetMeCreateIoT as an app.

### CODE REVIEW

**Comment by GonZo** [comment]
I'm curious to understand if this is the best way of doing it, with submodules.  

UPDATE :
Seems that this is not going to be upstreamed as @mtusnio told me.

File: .gitmodules [hunk 1]
```diff
@@ -10,3 +10,6 @@
 [submodule "dev/ca8210"]
 	path = dev/ca8210
 	url = https://github.com/Cascoda/ca8210-contiki.git
+[submodule "apps/letmecreateiot"]
```
Comment (manohar-narkhede-imgtec) [inline .gitmodules]: As it is specific to Mikro-e platform, it can be downloaded at platform/mikro-e/apps/letmecreateiot.

**Review by manohar-narkhede-imgtec — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (mtusnio) [reply .gitmodules]: Makes sense, I'll move it & update the app to make sure everything builds.

**Review by manohar-narkhede-imgtec — APPROVED** [review]

**Comment by manohar-narkhede-imgtec** [comment]
Do we need to add compilation of some of the examples in letmecreateIot in Jenkinsfile also?

**Comment by mtusnio** [comment]
It'd make sense to kick LetMeCreateIoT builds whenever we are merging into Creator, however that would require switching LMCIoT CI into a Jenkinsfile and all that. I'm not sure if it's part of this task, I think it's a wider "redo LMCIoT CI" thing.

**Pull request merged**